test_runner/regress/test_branching.py

```diff
@@ -251,7 +259,7 @@ def test_competing_branchings_from_loading_race_to_ok_or_err(neon_env_builder: N
         ".*request{method=POST path=/v1/tenant/.*/timeline request_id=.*}: request was dropped before completing.*"
     )
     env.pageserver.allowed_errors.append(
-        ".*Error processing HTTP request: InternalServerError\\(Timeline .*/.* already exists in pageserver's memory"
+        ".*Error processing HTTP request: InternalServerError.+Already exists.*"
```

Comment:
Did you see if we have a test case which would show that the response http code does not change because of these changes? At least the cases I've added near this comment do not assert that.